src/Context/cartContext.js

```diff
@@ -0,0 +1,42 @@
+import { createContext, useReducer, useContext } from "react";
+
+const CartContext = createContext();
+
+const cartFunction = (cartState, action) => {
+    switch(action.type){
```

Comment:
this can be kept at a separate reducer function, will make the code more readable

Reply:
Okay, will do it.